### DISCUSSION TITLE
how to download Chromebook apps like GeoGebra if district blocked downloading chrome apps

### DISCUSSION THREAD

**Discussion by LieutenantGst** [3kh0/ext-remover#1015] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
my school has blocked downloading chrome apps how to fix for V123

**Comment by Blobby-Boi** [comment, marked as answer]
you dont :)
use another webview

**Reply by Blobby-Boi** [reply]
List of apps with webviews:
SparkVUE
Geogebra Classic
S2Bot4Scratch
McMillan learning ebook
Droneblocks
myHomework student planner
3Dview
Coding with chrome
Mackinvia
Ipevo Whiteboard
TestNav

**Reply by LieutenantGst** [reply]
![Screenshot 2024-04-15 3 11 15 PM](https://github.com/3kh0/ext-remover/assets/145707265/8e798321-0ef7-447a-987b-1400e7a9000b)
No what I mean is like every time I try to download a chrome app it says blocked by admin how could I fix this I want to do a WebView exploit

**Reply by Blobby-Boi** [reply]
its not EVERY app
your school probably has a policy that whitelists VERY FEW apps and you havent stumpled across one yet
try going to chrome://policy and search for "extensioninstallallowlist" to see which ones they do allow

**Reply by bananstache** [reply]
Another extension with webview is Read&Write for Google Chrome. It uses webview when on the microsoft login page, from there you can click login with Github then search up google and click google.com.

**Reply by Blobby-Boi** [reply]
True, but it’s patched in new read&write versions and in chromeos v115.

**Reply by LieutenantGst** [reply]
hey @Blobby-Boi so what if my school admin blocked downloading every extension and app for the Chromebook is there any other way

**Reply by Blobby-Boi** [reply]
they did? if you search in chrome://policy for extensioninstallallowlist does nothing popup?

**Reply by LieutenantGst** [reply]
Yeah @Blobby-Boi it said only the ones the school downloaded so it’s securly and then one that my admin haven’t blocked which I uorgin which nothing works for that then one that I can’t download which is the stupid date testing chrome app but yeah I can’t download any chrome app for webveiw

**Reply by Blobby-Boi** [reply]
then try going to chrome://chrome-signin

**Reply by LieutenantGst** [reply]
@Blobby-Boi so how do I use a email for the chrome://chrome-signin

**Reply by Blobby-Boi** [reply]
go to chrome://chrome-signin
type spstudent1@d11.org
click more options
click signin with github
click forgot password
click the github logo at the top
search "google"
click "google.com"

**Reply by LieutenantGst** [reply]
@Blobby-Boi where is the more options I see the next option then it says "this account is not allowed to sign in with this network please help

**Reply by Blobby-Boi** [reply]
it should be at the bottom of the screen
![image](https://github.com/3kh0/ext-remover/assets/131911782/9a3df4d2-56ab-49e8-b2e1-48f8434f15e8)
it says signin options

**Reply by LieutenantGst** [reply]
@Blobby-Boi when I go to the chrome://signin I see this 
![Screenshot 2024-04-18 8 43 11 AM](https://github.com/3kh0/ext-remover/assets/145707265/74ae917f-80de-408a-8ffa-4f962807f8b3)
please help me get to that page

**Reply by AshtonDavies** [reply]
> @Blobby-Boi when I go to the chrome://signin I see this 
> ![Screenshot 2024-04-18 8 43 11 AM](https://github.com/3kh0/ext-remover/assets/145707265/74ae917f-80de-408a-8ffa-4f962807f8b3)
> please help me get to that page

Type in _spstudent1@d11.org_, then more options.

Scroll up to @Blobby-Boi 's instructions reply for full steps.

**Reply by LieutenantGst** [reply]
@AshtonDavies when I try to log in it says that I can’t use this account on this network

**Reply by CGRmags** [reply]
I think theres antoehr but i forgot :(

**Reply by CGRmags** [reply]
Mines just ablack screen when i see put that into the search bar

**Reply by LieutenantGst** [reply]
@CGRmags you click next then it should show the log in screen but I can’t use that email to get the use GitHub login

**Reply by AshtonDavies** [reply]
> 
> @AshtonDavies when I try to log in it says that I can’t use this account on this network 

Probably doesn't work with blocked accounts then.

**Reply by Blobby-Boi** [reply]
you dont have to login to spstudent1@d11.org

you have to click Sign-in Options

**Reply by AshtonDavies** [reply]
> you dont have to login to spstudent1@d11.org
> 
> you have to click Sign-in Options

Google's sign-in system does not have "More Options." That is Microsoft, which is probably connected to Google accounts in your district.

Entering a Microsoft-connected email should take you to their screen within chrome://signin, but outside accounts are usually blocked.

**Reply by LieutenantGst** [reply]
hey @AshtonDavies and @Blobby-Boi so I went to the Microsoft login and pressed more options then login with GitHub I clicked forget password and clicked the GitHub logo but no webveiw

**Reply by Blobby-Boi** [reply]
Are you searching “google” in GitHub?

**Reply by LieutenantGst** [reply]
I searched Google in the GitHub but that GitHub page wasn’t a webveiw only put me into a new web page in my browser

**Reply by Blobby-Boi** [reply]
Is the Microsoft login page a webview? If it is I know an alternative that doesn’t require GitHub since GitHub is not a webview

**Reply by LieutenantGst** [reply]
@Blobby-Boi can you tell me the alternative please

**Reply by Blobby-Boi** [reply]
In the microsoft signin type "e@e.com" where it says password click forgot password and then click "I dont have any of these"
on the top left click the microsoft logo then click the search icon and search "bingapp.microsoft.com" then click the first result and then click "Downlaod on the appstore". Scroll all the way down and click terms of use. Then scroll all the way down again and on the left click "Apple bag check class action settlement". Then click the AppleBagCheckSettlement.com link. Scroll all the way down and click privacy policy. Go through that document until you see two links that say here. Click the second one. Scroll all the way down and click google. 

this is so long lol I spent like 10 minutes writing it

**Reply by myschoolisass** [reply]
you can also do what ^ said by going to sitemap (at the bottom), and scrolling down until you see bing, and clicking on it! (follow what he says until you reach "click on search icon". if bing is blocked, do not use.

**Reply by LieutenantGst** [reply]
hey @Blobby-Boi and @myschoolisass I don't get the WebView I did the entire process but also what's the sitemap?

**Reply by Blobby-Boi** [reply]
wdym you dont get the webview

**Reply by LieutenantGst** [reply]
@Blobby-Boi I don't have a web view popup I thought you had an alternative for another web view

**Reply by Blobby-Boi** [reply]
doesnt chrome://chrome-signin work as a webview?
If not I already gave you a list of apps but here it is again
SparkVUE
Geogebra Classic
S2Bot4Scratch
McMillan learning ebook
Droneblocks
myHomework student planner
3Dview
Coding with chrome
Mackinvia
Ipevo Whiteboard
TestNav

**Reply by jeffrey6576i** [reply]
how do i do the geogebra one

**Reply by schoolexploitkid** [reply]
chrome://chrome-signin is patched for me, the github just opens in new tab, idk if there are alternate instructions.

**Reply by jeffrey6576i** [reply]
i think its patched in all version @schoolexploitkid

**Reply by artisticmclaren** [reply]
> i think its patched in all version @schoolexploitkid

I have it working on v130 by clicking Docs, then Pricing then searching from there

**Reply by jeffrey6576i** [reply]
im on v130 too? howw
